coalahtml/_coalahtml/index.html

```diff
@@ -9,9 +9,9 @@
     <link rel='stylesheet' type='text/css'
       href='http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext'>
     <link rel="stylesheet"
-          href="bower_components/bootstrap/dist/css/bootstrap.css">
+    href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
```

Comment:
@hypothesist  Nice. I wasn't aware of that. Thanks. I'll make the changes *if*  we fetch files from cdn rather than having them saved offline. :D